iptstate.cc

```diff
@@ -1260,14 +1272,8 @@ void build_table(flags_t &flags, const filters_t &filters, vector<tentry_t*>
    * Initialization
    */
   // Nuke the tentry_t's we made before deleting the vector of pointers
```

Comment:
wanna move this comment up to where you put the code? :)

Reply:
Sure